.textlintrc

```diff
@@ -38,36 +38,72 @@ rules:
   spelling:
     language: en
     skipPatterns:
-    - '/placeos/i'
+    # for anything in terminology, add only the corrected version
+    - '/[0-9]+/g'             # numbers 
+    - '/\b(?:[A-Z]){2,}\b/g'  # acronyms
+    - '/\b\S*\.\S{2,4}/g'     # filenames
+    - '/analytics/i'
+    - '/automations/i'
     - '/backoffice/i'
-    - '/[A-Z]{2,}/'      # acronyms
-    - '/[0-9]+/g'        # numbers 
+    - '/(re)?balance(r)?/i'
+    - '/integrat(ion|or)[s]?/i'
+    - '/microservice[s]?/i'
+    - '/placeos/i'
```

Comment:
This is not appear downcased.

Reply:
Yep, good catch. I think that was in from an earlier use of it in things like variable names, URIs etc, but those are now skipped appropriately